app/controllers/consumption/testOnly/ListRetrievalController.scala

```diff
@@ -42,14 +42,90 @@ class ListRetrievalController @Inject() (
           Future.successful(
             Ok(Json.obj("data" -> Json.toJson(listRetrievalService.getCustomsOffice)))
           )
+        case "CountryCodesFullList" =>
+          Future.successful(
```

Comment:
Very minor but could remove the `async` from the action then wouldn't need to wrap everything in a `Future.successful`

Reply:
Yes. Some of this was a bit of a rush. Will do this. 👍